plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/tasks/monitor_incident_events/task.py

```diff
@@ -57,7 +57,7 @@ def remove_old_and_sort_incident_events(
         cleaned_events_after_cutoff.sort(key=lambda x: x.get(time_sorting_field, -1))
```

Comment:
Not sure if this `monitor_incident_events` task is being used, should it be removed from the plugin?

Reply:
Unsure, keeping it in until the end we work out if it is being used or not

Reply:
> Not sure if this `monitor_incident_events` task is being used, should it be removed from the plugin?

It is not being used. We decided to create our task from scratch so I will delete it to resolve any confusion